plugins/gatsby-source-jenkinsplugins/utils.js

```diff
@@ -159,6 +161,26 @@ const fetchPluginData = async ({createNode, reporter}) => {
         page = pluginsContainer.page + 1;
     } while (!page || pluginsContainer.page < pluginsContainer.pages);
     await Promise.all(promises);
+    const updateUrl = 'https://updates.jenkins.io/current/update-center.actual.json';
```

Comment:
```suggestion
    const updateUrl = 'https://updates.jenkins.io/update-center.actual.json';
```

Implementation detail that should not be relied on.